src/scripts/train/ladder/olmo3_ladder.py

```diff
@@ -0,0 +1,387 @@
+import argparse
+import logging
+import sys
+import textwrap
+from pathlib import Path
+
+import rich
+
+import olmo_core.io as io
+from olmo_core.data import DataMix, TokenizerConfig
+from olmo_core.data.composable import *
+from olmo_core.exceptions import OLMoConfigurationError
+from olmo_core.internal.common import build_launch_config, get_gpu_type, get_root_dir
+from olmo_core.launch.beaker import (
+    BeakerLaunchConfig,
+    BeakerPriority,
+    OLMoCoreBeakerImage,
+)
+from olmo_core.model_ladder2 import *
+from olmo_core.utils import prepare_cli_environment
+
+log = logging.getLogger(__name__)
+
+
+def parse_args() -> argparse.Namespace:
+    command_descriptions = {
+        "dry_run": "Simulate the ladder run for a given model size, showing the LR schedule plot.",
+        "benchmark": textwrap.dedent(
+            """
+            Run a benchmark for a given model size locally.
+            This usually isn't invoked directly, but rather via `launch_benchmark`.
+            """
+        ).replace("\n", " "),
+        "launch_benchmark": "Launch a Beaker job to run a benchmark for a given model size.",
+        "run": textwrap.dedent(
+            """
+            Run the ladder for a given model size locally.
+            This usually isn't invoked directly, but rather via `launch_run`.
+            """
+        ).replace("\n", " "),
+        "launch_run": "Launch a Beaker job to run the ladder for a given model size.",
```

Comment:
Could we have a utility for launching a set of runs? For example all of the rungs of size less than or equal to the specified size? I see myself kicking off the bottom 3/4 sizes all at once pretty frequently when evaluating ideas.